### PR TITLE
Copy refinements.

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
Addressing more of the issues in #1034

### CODE REVIEW

File: locales/en/app.ftl [hunk 1]
```diff
@@ -225,8 +228,8 @@ how-fxm-3-blurb = If you use {-brand-name}, you’ll receive a notification if y
   site that’s been breached. Find out right away if you were part of that breach 
   and what you can do about it.
 
-wtd-after-website = What to do after a website breach
-wtd-after-data-agg = What to do after a data aggregator breach
+wtd-after-website = What to do after a website breach:
```
Comment (flodolo) [inline locales/en/app.ftl]: Do we care about other localizations picking up this change?

File: locales/en/app.ftl [hunk 2]
```diff
@@ -33,6 +34,8 @@ scan-title = {-product-name} : Scan Results
 
 user-add-invalid-email = Invalid Email
 user-add-email-verify-subject = Verify your subscription to {-product-name}.
+user-add-duplicate-email = This email has already been added to {-product-name}.
+user-add-duplicate-email-part-2 = Visit your { $preferencesLink } to check the status of { $userEmail }.
```
Comment (flodolo) [inline locales/en/app.ftl]: What is replacing `$preferencesLink`? Please add a comment

File: locales/en/app.ftl (hunk 1 above)
Comment (lesleyjanenorton) [reply locales/en/app.ftl]: What are your thoughts on this? If it doesn't change much in other locales, then I suppose not?

Comment (flodolo) [reply locales/en/app.ftl]: Let's change it without new IDs. After all, they are supposed to look at the website and decide if it works or not for their language.

**Comment by groovecoder** [comment]
The Travis failures are un-related to these changes (and are fixed in latest master). Merging.

**Comment by groovecoder** [comment]
Oh f. I overlooked @flodolo's comments in my hurry to merge this before going out on PTO.

**Comment by flodolo** [comment]
At this point it's only missing a comment. The sooner we fix it, the better.

**Pull request merged**